README.md

```diff
@@ -2,133 +2,54 @@
-| WebFinger | <kbd>search</kbd> |
-| RDF | <kbd>search</kbd> <kbd>fetch</kbd> |
-| Bitcoin (e.g. sending signed transactions to a node) | *(to be evaluated)* <kbd>send</kbd> <kbd>receive</kbd> ... |
-| Ethereum (e.g. contract events) | *(to be evaluated)* |
-| WebRTC (signaling) | *(to be evaluated)* |
-| Signal | *(to be evaluated)* |
-| [MicroPub](https://indieweb.org/micropub) (IndieWeb, ...) | *(to be evaluated)* |
-| [OStatus](https://en.wikipedia.org/wiki/OStatus) (GNUSocial, Mastodon, ...) | *(to be evaluated)* |
+### Dependencies
 
-# Setup
+```$ yarn run install```
 
-`$ yarn install`
+### Tests
 
-# Running
+```$ yarn run test```
 
-For development purposes, with examples enabled, run:
+### Run
 
-`$ DEBUG=sockethub* yarn run dev`
+```$ yarn run start```
 
-You should then be able to browse to `http://localhost:10550` and try out the examples.
+Or, for debugging or development:
 
-For production, with examples disabled.
+```$ yarn run dev```
 
-`$ DEBUG=sockethub* yarn run start`
+## Packages
 
-# Running using Docker Compose
+Packages maintained in this repository.
 
-Requires [Docker Compose](https://docs.docker.com/compose/) 1.10.0+
-
-`$ docker-compose up`
-
-> If you’re using Docker natively on Linux, Docker for Mac, or Docker for
-> Windows, then sockethub should now be listening on port 10550 on your Docker
-> daemon host. Point your web browser to http://localhost:10550 to find
-> sockethub. If this doesn’t resolve, you can also try
-> http://0.0.0.0:10550.
-
-> If you’re using Docker Machine on a Mac or Windows, use docker-machine ip
-> MACHINE_VM to get the IP address of your Docker host. Then, open
-> http://MACHINE_VM_IP:10550 in a browser.
-
-# Environment Variables
-
-* PORT
-Defaults to `10550`
-* HOST
-Defaults to `localhost`
-* DEBUG
-Specify the namespace to console log, ie. `sockethub*` will print all sockethub related debug statements, whereas `*` will also print any other modules debug statements that use the `debug` module.
-
-* REDIS_PORT
-Defaults to `6379`
-* REDIS_HOST
-Defaults to `localhost`
-
-***OR***
-
-* REDIS_URL
-Overrides `REDIS_HOST` and `REDIS_PORT`, can specify a full redis connect URL (eq. `redis://username:password@host:port`)
-
-# Command-line params
-```
-  --help       : this help screen
-  --info       : displays some basic runtime info
-
-  --examples   : enabled examples page and serves helper files like jquery
-
-  --host       : hostname to bind to
-  --port       : port to bind to
-```
+* [packages/activity-streams.js](packages/activity-streams.js)
+* [packages/irc2as](packages/irc2as)
+* [packages/sockethub](packages/sockethub)
+* [packages/sockethub-platform-dummy](packages/sockethub-platform-dummy)
+* [packages/sockethub-platform-feeds](packages/sockethub-platform-feeds)
+* [packages/sockethub-platform-irc](packages/sockethub-platform-irc)
+* [packages/sockethub-platform-xmpp](packages/sockethub-platform-xmpp)
+* [packages/sockethub-schemas](packages/sockethub-schemas)
 
 # License
```

Comment:
The other headings have been moved to level 2 (which I think is much better), so License and Credits should be changed to level 2 as well.